File: 02_mar/numVals/numVals.js
function getCount(countObj) {
	var count = 0;

	for (var key in countObj) {
		count++;
	}

	return count;
}

var inpObj = {
	"band": "Travis Shredd & the Good 'ol Homeboys",
	"style": "Country/Metal/Rap",
	"album": "The Neighbor of the Beast"
};

console.log(getCount(inpObj));

File: 02_mar/numVals/numVals.py
def get_length(count_dict):
	count = 0

	for key in count_dict:
		count += 1

	return count


inp_dict = {
	"band": "Travis Shredd & the Good 'ol Homeboys",
	"style": "Country/Metal/Rap",
	"album": "The Neighbor of the Beast"
}

print get_length(inp_dict)
